get_issue_metadata.py

```diff
@@ -0,0 +1,107 @@
+import time
+
+from datetime import datetime as dt
+
+import check_rate_limit as crl
```

Comment:
Begin with `from __future__ import absolute_import`, and then change this to `import .check_rate_limit` (note the leading period).